### ISSUE TITLE
Can't compile with GPU=1

### ISSUE THREAD

**Issue by AnanasPie** [pjreddie/darknet#262]
Hi,

According to the following https://github.com/pjreddie/darknet/issues/230
now I get another compilation error:
![image](https://user-images.githubusercontent.com/7532861/31861689-3be3ac40-b73a-11e7-82c7-e7f8ecb93be1.png)

**Comment by anujonthemove** [comment]
Hey, 
Could you please share more details here?
I don't see any problems compiling it on a system with the following specifications:

`Ubuntu 16.04
CUDA v8
cuDNN v5.1 for CUDA 8.0
OpenCV 3.3.0
Nvidia GeForce 940MX (4GB)`

Flags set in Makefile:

**Comment by jrriosBC** [comment]
I have the same issue

**Comment by anujonthemove** [comment]
Windows?

**Comment by jrriosBC** [comment]
No, the issue was on Ubuntu 16.04 but I fixed it by downgrading to CUDA 8.0 and CUDNN 6.0 as suggested by this person https://github.com/pjreddie/darknet/issues/230#issuecomment-335592510
